lib/foreman/renderer/scope/macros/host_template.rb

```diff
@@ -121,6 +121,57 @@ def ks_console
             (@port && @baud) ? "console=ttyS#{@port},#{@baud}" : ''
           end
 
+          apipie :method, 'Generates commands for package(s) installation' do
+            description 'Supported OS families: Redhat, Debian & Suse'
+            required :packages, String, desc: "Package(s) to install"
+            returns String, desc: 'Installation commands'
+            example "CentOS host: install_packages('pkg1') #=> yum -y install pkg1"
+            example "Fedora host: install_packages('pkg1 pkg2') #=> dnf -y install pkg1 pkg2"
+          end
+          def install_packages(packages)
+            return '' if packages.blank?
+
+            banner = <<~CMD
+              echo '#'
+              echo '# Installing packages'
+              echo '#'
+            CMD
+
+            case host.operatingsystem&.family
+            when 'Redhat'
+              fedora = host.operatingsystem.name.downcase == 'fedora'
+              rhel_dnf = host.operatingsystem.name.downcase == 'redhat' && host.operatingsystem.major.to_i >= 8
```

Comment:
in fact the same applies to centos and probably future clones (alma, rocky, whatever), I guess we should use OS family == redhat && major > 8

the condition below would be fedora || redhat_family_dnf